test/containerized_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import json
+import http.client as http_client
+
+import pytest
+
+from ovirt_imageio._internal import config
+from ovirt_imageio._internal import server
+from ovirt_imageio._internal.units import KiB
+
+from . import testutil
+from . import http
+
+
+TICKET_IMG_UUID = "file"
+TICKET_IMG_SIZE = 16 * KiB
+
+
+@pytest.fixture(scope="module")
```

Comment:
Why do we need module scope? we have one test and starting it is pretty fast. Using the same ticket, image and server for all tests is not a good idea.

Reply:
I used the module scope so that it could be used from the server fixture, that had module scope as other tests do.

If it is ok to have new server per test, I'll remove all module fixtures and move to tmpdir instead of the factory.